analysis/tests/src/expected/TypeContextCompletion.res.txt

```diff
@@ -0,0 +1,12 @@
+Complete src/TypeContextCompletion.res 10:44
+posCursor:[10:44] posNoWhite:[10:43] Found expr:[10:11->18:1]
+Pexp_apply ...[10:11->10:30] (~someVariant10:32->10:43=...[13:0->17:3])
+Completable: CnamedArg(Value[someVariantToString], "", [someVariant])
+Found type for function (~someVariant: someVariant) => string
```

Comment:
Need to check where this is coming from: presumably named arg completion finding the type of the function.